Add examples: CSS initial and inherit (#43309)
mdn/content: files/en-us/web/css/reference/values/initial/index.md

## Examples

### Basic usage

In this example, we use the `initial` value to reset {{cssxref("color")}} and {{cssxref("font-size")}} for an element.

#### HTML

```html
<p>
  This text is red and large.
  <em
    >This text is in the initial color (typically black) and initial size
    (typically 16px).</em
  >
  This is red and large again.
</p>
```

#### CSS

> We define a `color` and `font-size` on the paragraph, and reset the values on the {{htmlelement("em")}} by setting these property values to `initial`.

Reviewer: slight tweak
```suggestion
We set `color` and `font-size` on the `<p>` element, then set those properties to `initial` on the {{htmlelement("em")}} element to reset them.
```

```css
p {
  color: red;
  font-size: 2rem;
}

em {
  color: initial;
  font-size: initial;
}
```

#### Result

{{EmbedLiveSample('Using_initial_to_reset_color_for_an_element')}}

> With the `initial` keyword in this example, the `color` and `font-size` values on the `em` element are restored to the initial values of [`color`](/en-US/docs/Web/CSS/Reference/Properties/color#formal_definition) and [`font-size`](/en-US/docs/Web/CSS/Reference/Properties/color#formal_definition), respectively, as defined in the specification.

Reviewer: `font-size` link needs to be updated
